### DISCUSSION TITLE
Warning message in console

### DISCUSSION THREAD

**Discussion by niehaus-alexander** [neuefische/questions#843] [AWS] [answered]
We have the following warning in our console:

**_Warning: Prop `style` did not match. Server: "null" Client: "border:"[...]_**

We think it has to do with an inline styling we use to highlight a button.
This is the [link](https://github.com/Braun-Johannes/Capstone-Project-BattlePets/tree/feature/edit) to the current state of our project.

In the components/PetList.jsx on line 13 we use inline styling and to change the border color of the rendered button based on whether the selectedPet equals the pet of the map methods iteration.

We think the warning might have to do with the fact the overwriting of the styled component while selected Pet is not defined when the App
is rendered for the first time. Any ideas for another approach?

**Comment by doemser** [comment, marked as answer]
Try not using inline-styles, instead use a prop

```javascript
<StyledListItem $selected={selectedPet === pet}>...</StyledListItem>
```

**Reply by niehaus-alexander** [reply]
It worked, thank you!
